e2e/wdio/core/fixtures/appData/object-status-contents.ts

```diff
@@ -0,0 +1,72 @@
+const statusAttr = 'status';
+const colorAttr = 'color';
+const backgroundColorAttr = 'background-color';
+const indicatorAttr = 'ng-reflect-indication-color';
+const sizeAttr = 'ng-reflect-large';
+const semanticColors = {
+    negative: '187,0,0',
+    critical: '233,115,12',
+    positive: '16,126,62',
+    informative: '5,59,112',
+    null: '106,109,112'
+};
+const genericColors = {
+    1: '136,0,0',
```

Comment:
How it is used in my test, 
There is an attribute in the selector which says which color and they use number values for these colors. color '1' = rbg(136,0,0) 
So I have the function that gets the number of the color and uses it to select the correct color from the genericColors obj. 
This way, order of the array is not important because the color will be chosen based on the selector attribute.

Reply:
please convert to array and it looks good to me